.github/workflows/stress.yml

```diff
@@ -111,15 +113,25 @@ jobs:
       run: scripts/prepare-machine.ps1 -Tls ${{ matrix.vec.tls }} -ForTest ${{ matrix.vec.xdp }}
       shell: pwsh
     - name: spinquic (PR)
-      if: github.event_name == 'pull_request'
+      if: (!(matrix.vec.os == 'ubuntu-22.04' && matrix.vec.xdp == '-UseXdp') && github.event_name == 'pull_request')
```

Comment:
Why are you excluding XDP?

Reply:
Oh, I see. You're adding `sudo` below. Is that _really_ necessary?

Reply:
xdp requires sudo.
So the if-else is needed to add sudo only for ubuntu xdp

Reply:
oh, found the one!!!
libxdp_set_print(NULL);
libbpf_set_print(NULL);